src/questions/testing-questions.md

```diff
@@ -9,3 +9,6 @@ permalink: /questions/testing-questions/index.html
 * What is the difference between a unit test and a functional/integration test?
 * What is the purpose of a code style linting tool?
 * What are some of the testing best practices?
+* What is an end to end test, and how is it different from a unit or integration test?
```

Comment:
`* What is the difference between a unit test and a functional/integration test?` 
is it a dublicate?

Reply:
Yes, this is. That question can be deleted.

Reply:
so ? are you going to do that?